docs/tips-tricks.md

```diff
@@ -22,3 +22,10 @@ When the **headlessServiceEnabled** option is enabled (true) in your KafkaCluste
 
 When the **headlessServiceEnabled** option is disabled (false), it creates a ClusterIP service. When using a ClusterIP service, your client application doesn’t need to be aware of every Kafka broker endpoint, it simply connects to *kafka-all-broker:29092* which covers dynamically all the available brokers. That way if the Kafka cluster is scaled dynamically, there is no need to reconfigure the client applications.
 
+## Retrieving broker configuration during downscale running
```

Comment:
```suggestion
## Retrieving broker configuration during downscale is running
```

Reply:
This one sounds weird to me, your earlier suggestion would be best I think.